### DISCUSSION TITLE
Using the {FolderPath} parameter in custom prompts

### DISCUSSION THREAD

**Discussion by huangzuomin** [logancyang/obsidian-copilot#555] [Q&A]
Hi everyone,

I'm trying to use Obsidian Copilot to summarize the files in the {FolderPath:test} folder. However, when I use the following prompt, I get the following warning:

Warning: No valid notes found for the providedpath 'FolderPath:test.

I'm not sure how to use the {FolderPath} parameter correctly. Can anyone provide some assistance?

Things I've tried:

Making sure that the {FolderPath:test} folder exists and contains files.
Trying different paths, such as /Users/myusername/test.
Disabling and re-enabling the Obsidian Copilot plugin.

I still can't get it to work. Any help would be greatly appreciated.

**Comment by hb-man** [comment]
Just for history, and in case you didn't figure it out: 
you put the actual folder path in curly brackets, so the folder `test` assuming it's in the root of the vault, the correct usage would be `{test}`, not `{FolderPath:test}`
